lcu_integration.py

```diff
@@ -21,21 +21,21 @@
 # LCU logic taken from https://github.com/elliejs/Willump
 # We want to implement a synchronous approach,
 # so we are not using the library.
-def get_lcu_process():
+def get_lcu_process() -> Process | None:
     """
     Get the LeagueClientUx process, aka. the process of the League client.
 
     Returns:
         The process if found, else None.
 
     """
-    for process in process_iter():
-        if process.name() in {"LeagueClientUx.exe", "LeagueClientUx"}:
-            return process
-    return None
+    return next(
+        (process for process in process_iter() if process.name() in {"LeagueClientUx.exe", "LeagueClientUx"}),
+        None,
+    )
```

Comment:
I personally find this less readable, but could be swayed. @Gadsee, thoughts?

Reply:
Typically, list comprehensions save space, and I'm all for that, but it doesn't save us anything in this case. So readability is the only thing to consider, which I agree with Kyrluckechuck, is slightly worse here.